### PR TITLE
Seth staging

### PR DESCRIPTION
**Pull request by 20c33406**
Added home page

### CODE REVIEW

**Review by Gerald12344 — APPROVED** [review]
My changes are good!

**Comment by Gerald12344** [comment]
Oh yeah and the rest of the changes LGTM

File: components/global/MainCard.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Box, Button, Card, Group, Text } from '@mantine/core';
+
+export default function MainCard() {
```
Comment (jw2476) [inline components/global/MainCard.tsx:3]: I hate this so much but apparently this is okay :/

File: components/homepage/homepageGrid.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    children: any; // React children (inside brackets)
+    Title: string;
+    Description: string;
+    colour: {
+        from: string;
+        to: string;
+    };
+}
+
+export default function HomepageGrid({
```
Comment (jw2476) [inline components/homepage/homepageGrid.tsx:14]: This looks like a great way to do this

File: pages/index.tsx [hunk 3]
```diff
@@ -1,18 +1,113 @@
...
+        <AppShell
+            styles={(theme) => ({
+                main: {
+                    backgroundColor: theme.colorScheme === 'dark' ? theme.colors.dark[8] : theme.colors.gray[0],
+                },
+            })}
+            header={<Navbar />}
+        >
+            <MainCard />
+            <SimpleGrid cols={3} mt="md">
```
Comment (jw2476) [inline pages/index.tsx:31]: Just a note that all these simple grid tags can be merged into one, also not sure how this will function on mobile

**Review by jw2476 — APPROVED** [review]
A couple of optimizations could be done on the HTML, starting to understand React now... Not sure how this will look on mobile but we can fix that later, first PR approved!

**Pull request merged**